src/main/resources/database/data.sql

```diff
@@ -34,15 +35,17 @@ INSERT INTO `position` (p_title, pinned_to_comment)
 VALUES ('Завхоз', false);
 
 INSERT INTO `user` (u_login, u_password, position_id)
-VALUES ('admin', '$2a$10$yKj28aFTziVYvmegHoSLy.WZVPJINyt8xCKZDhSPfxu2j7dtAFtRe', 1);
+VALUES ('admin', @PWD, 1);
 INSERT INTO `user` (u_login, u_password, position_id)
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 5 times. ([plsql:S1192](https://sonarcloud.io/organizations/final-project/rules#rule_key=plsql%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=180&issues=AWb5WfX4W3Y69bz5BsNk&open=AWb5WfX4W3Y69bz5BsNk)

[ISSUE_KEY]: AWb5WfX4W3Y69bz5BsNk